rbac/management/tenant/model.py

```diff
@@ -0,0 +1,327 @@
+"""Additional tenant-related models."""
+
+import uuid
+from typing import List, Optional
+from typing import NamedTuple
+
+from django.conf import settings
+from django.db import models, transaction
+from kessel.relations.v1beta1.common_pb2 import Relationship
+from management.principal.model import Principal
+from management.role.relation_api_dual_write_handler import RelationReplicator, ReplicationEvent, ReplicationEventType
+from management.workspace.model import Workspace
+from migration_tool.utils import create_relationship
+
+from api.models import Tenant, User
+
+
+class TenantMapping(models.Model):
+    """Tenant mappings to V2 domain concepts."""
+
+    tenant = models.OneToOneField(Tenant, on_delete=models.CASCADE)
+    default_group_uuid = models.UUIDField(default=uuid.uuid4, editable=False, null=False)
```

Comment:
Do we need to new uuids for default_group and default_admin_group ? Is it possible to reuse existing ones from RBAC db  for ?

Reply:
We can't because these do not correspond with RBAC groups initially. The RBAC default groups are shared across the entire platform. These need to be per-tenant. So, we need to have a separate UUID for these, and then reuse the same UUID for custom default group so that the group membership lines up for custom default group rules.